bundles/action/org.openhab.action.pushsafer/src/main/java/org/openhab/action/pushsafer/internal/Pushsafer.java

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
```

Comment:
The new copyright header blob should be used. It's [here.](https://github.com/openhab/openhab/blob/master/src/etc/header.txt)